spec/controllers/applications_controller_spec.rb

```diff
@@ -186,6 +197,19 @@
           end
         end
 
+        context "missing optional email for Google" do
+          let(:email_for_google) { "" }
+
+          it "should add an notice to the flash" do
```

Comment:
s/an/a/

Reply:
Good catch, I copied a typo -- fixed this instance and the other instance in this file.